sys/abi/src/lib.rs

```diff
@@ -625,17 +639,17 @@ impl core::convert::TryFrom<u16> for Kipcnum {
 #[repr(C)]
 #[derive(Default, Copy, Clone, Debug, FromBytes, AsBytes)]
 pub struct SAUEntry {
-    pub rbar: u32,
-    pub rlar: u32,
+    pub rbar: AbiSize,
+    pub rlar: AbiSize,
 }
 
-pub const HEADER_MAGIC: u32 = 0x1535_6637;
+pub const HEADER_MAGIC: AbiSize = 0x1535_6637;
 
 #[repr(C)]
-#[derive(Default, AsBytes, FromBytes)]
+#[derive(Default, Copy, Clone, AsBytes, FromBytes)]
```

Comment:
How come Copy and Clone traits were needed here when everything else was s/32/AbiSize/ ?

Reply:
Oops. I had this removed in my local repo, but somehow never ended up adding to the patch. These are not required with AbiSize or u32. Fixed in latest patchset.